source/Nevermore/RelationalJsonContractResolver.cs

```diff
@@ -16,9 +16,20 @@ public RelationalJsonContractResolver(IRelationalStoreConfiguration configuratio
             this.configuration = configuration;
         }
 
-        protected override JsonProperty CreateProperty(MemberInfo member, MemberSerialization memberSerialization)
+        protected override IList<JsonProperty> CreateProperties(Type type, MemberSerialization memberSerialization)
```

Comment:
This is the base implementation, should we copy that?

![image](https://user-images.githubusercontent.com/2888279/110421594-8ad6cb80-80e9-11eb-9ff5-314b2dd890c9.png)

Reply:
I did try that, but ran into an issue with the `GetNameTable` method. It's marked as `internal`, and returns the value of a private field. It looks like it's referenced a fair bit in the `JsonTextReader`, but all usages I could find have alternatives if the `JsonNameTable` doesn't yield any result.
I think it would be fine to exclude the name table related parts of the method if we intend to copy it. Thoughts?

Reply:
Calling JSON expert @slewis74

Reply:
Looking through the usages, the `JsonNameTable` appears to be a form of cache for property names. I would be confident in implementing this:
![Screen Shot 2021-03-11 at 10 14 52 am](https://user-images.githubusercontent.com/25342760/110716314-a0b3d000-8252-11eb-8973-4587ad3d4039.png)
It's as close to the original as we can make it